scripts/license_finder.sh

```diff
@@ -0,0 +1,3 @@
+curl https://raw.githubusercontent.com/bugsnag/license-audit/master/config/decision_files/global.yml -o config/global.yml
```

Comment:
If we're adding this to every pipeline, I think we should have a more robust way of including this file — if it moves or if the branch is renamed then this will break

Maybe we could build the config into a dockerfile that can be downloaded & used in each pipeline? Then any change to the config file can be automatically picked up, assuming everything points to the `latest` tag

Reply:
Agreed, there's a lot more we could be doing with the licence-audit tool going forwards.  For now I'm happy to take the risk that we need to updated all the pipelines in response to any changes in it (it's a low risk given what we're planning on doing).